src/com/amazon/corretto/crypto/provider/AmazonCorrettoCryptoProvider.java

```diff
@@ -555,6 +576,7 @@ public AmazonCorrettoCryptoProvider() {
 
     this.shouldRegisterAesCfb = (!isFips() || isExperimentalFips());
 
+    this.shouldRegisterMLKEM = Utils.isKemSupported();
```

Comment:
I think we need `!isFips()` here as well since we haven't submitted https://github.com/aws/aws-lc/commit/98500e8bc7dc3e3e5871519abf12cdc781ebe4e1 to the FIPS lab yet.